src/main/java/com/fasterxml/jackson/databind/util/EnumValues.java

```diff
@@ -125,19 +115,9 @@ public static EnumValues constructFromToString(MapperConfig<?> config, Annotated
         // build
         SerializableString[] textual = new SerializableString[enumConstants.length];
         for (int i = 0; i < enumConstants.length; i++) {
-            String name = names[i];
-            if (name == null) {
-                Enum<?> en = enumConstants[i];
-                name = en.toString();
-                // 01-Feb-2024, tatu: [databind#4355] Nulls not great but... let's
-                //   coerce into "" for backwards compatibility
-                if (name == null) {
-                    name = "";
-                }
-            }
-            if (useLowerCase) {
-                name = name.toLowerCase();
-            }
+            String enumToString = enumConstants[i].toString();
+            enumToString = (enumToString == null) ? "" : enumToString;
```

Comment:
Let's retain comment from above wrt [databind#4355], otherwise unclear how/why should we ever get `null` in here.

Reply:
Ah my bad. will do